tools/askor/askor_de.xml

```diff
@@ -0,0 +1,100 @@
+<tool id="askor_de" name="AskoR_DE" version="0.01">
+    <description>Edger for askomics </description>
+    <requirements>
+        <requirement type="package" version="3.34.6">bioconductor-limma</requirement>
+        <requirement type="package" version="1.4.29">r-statmod</requirement>
+        <requirement type="package" version="3.20.7">bioconductor-edger</requirement>
+        <requirement type="package" version="2.2.1">r-ggplot2</requirement>
+        <requirement type="package" version="1.1_2">r-rcolorbrewer</requirement>
+        <requirement type="package" version="0.7.0">r-ggrepel</requirement>
+        <requirement type="package" version="3.0.1">r-gplots</requirement>
+        <requirement type="package" version="1.3.2">r-optparse</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+      export ASKOR_PATH='$__tool_directory__' &&
+      Rscript '$__tool_directory__/Asko_DE.R'
+      --out ${out_pdf}
+      --outdir askor_output
+      --fileofcount ${fileofcounts}
+      --samples ${samples}
+      --contrasts ${contrasts}
+      --org ${organism}
+      --col_genes ${colgenes}
+      --sep ${sep}
+      --palette ${palette}
+      --select ${sel}
+      --remove ${remove}
+      --th_cpm ${th_cpm}
+      --rep ${rep}
+      --th_FDR ${fdr}
+      --normalization ${norm}
+      --adj ${adj}
+      --glm ${glm}
+      --nh ${nh}
+    ]]></command>
+    <inputs>
+        <param format="tsv" name="fileofcounts" type="data" label="File of counts" />
+        <param format="tsv" name="samples" type="data" label="descriptions of the samples" />
+        <param format="tsv" name="contrasts" type="data" label="descriptions of the contrasts" />
+        <section name="adv" title="Format options" expanded="false">
+          <param name="colgenes" type="integer" label="Column of the genes" value="1" optional="true" />
+          <param name="sep" type="text" label="field separator" value="\t" optional="true" />
+          <param name="palette" type="text" label="color palette (Rcolorbrewer)" value="Set2" optional="true" />
+          <param name="organism" type="text" label="organism" value="Asko" optional="true" />
+          <param name="nh" type="integer" label="number of genes in the heatmap" value="50" optional="true" />
+        </section>
+        <section name="selection" title="Samples selection options" expanded="false">
+          <param name="sel" type="text" label="list of selected samples (separated by comma)" value="" optional="true" />
```

Comment:
For optional params, you have to remove `value=""` and add a test in your command line. 
e.g.
 ```
#if $input
        -B "$input"
 #end if
```